prime-router/metadata/schemas/iPatientCare/iPatientCare-covid-19.schema

```diff
@@ -0,0 +1,323 @@
+# Base schema for iPatientCare clients
+# This schema is to be used to parse the standard lab report from iPatientCare
+# Created by Joel Biskie
+# Last Update: 07/28/2021
+# 
+name: ipatientcare-covid-19
+topic: covid-19
+description: iPatientCare CSV lab report schema
+basedOn: covid-19
+elements:
+
+#demographics
+    - name: patient_id
+      csvFields: [{name: MRN}]
+
+    - name: patient_last_name
+      csvFields: [{name: Last Name}]
+
+    - name: patient_middle_name
+      csvFields: [{name: Middle Name}]
+
+    - name: patient_first_name
+      csvFields: [{name: First Name}]
+
+    - name: patient_street
+      csvFields: [{name: Patient Address}]
+
+    - name: patient_city
+      csvFields: [{name: Patient City}]
+
+    - name: patient_state
+      csvFields: [{name: Patient State}]
+
+    - name: patient_zip_code
+      csvFields: [{name: Patient ZipCode}]
+
+    - name: patient_county
+      type: TABLE
+      table: zip-code-data
+      tableColumn: county
+      mapper: zipCodeToCounty(patient_zip_code)
+
+    - name: patient_phone_number
+      csvFields: [{name: Patient phone}]
+
+    - name: patient_dob
+      csvFields: [{name: Birth Date, format: "M/d/yyyy"}]
+
+    - name: patient_gender
+      csvFields: [{name: Sex}]
+
+# TODO - awaiting valid race values from iPatientCare
+    - name: patient_race
+      csvFields: [{name: RACE, format: $alt}]
+      altValues:
+        - code: 2106-3
+          display: White
+        - code: 1002-5
+          display: American Indian or Alaska Native
+        - code: 2028-9
+          display: Asian
+        - code: 2054-5
+          display: Black
+        - code: 2076-8
+          display: Native Hawaiian or Other Pacific Islander
+        - code: 2131-1
+          display: Other
+        - code: UNK
+          display: Unknown
+        - code: UNK
+          display: 
+        - code: ASKU
+          display: Asked, but unknown
+
+            
+# TODO - awaiting valid ethnicity values from iPatientCare
+    - name: patient_ethnicity
+      csvFields: [{name: Ethnicity, format: $alt}]
+      altValues:
+        - code: H
+          display: Hispanic or Latino
+        - code: N
+          display: Non Hispanic or Latino
+        - code: U
+          display: Patient Declines
+        - code: U
+          display: "NULL"
+        - code: U
+          display: WHITE    #????
+
+
+
+
+
+
+    - name: specimen_collection_date_time
```

Comment:
I believe you'll get format warnings (not errors) on these, as we're hoping to get timestamps.

Reply:
Ok - I haven't seen it warn.  We get just the one date from iPatientCare and have to use it everywhere.